### DISCUSSION TITLE
USB drive is only 8kB, cannot update

### DISCUSSION THREAD

**Discussion by viktak** [Ralim/IronOS#1785] [Q&A] [answered]
I have my TS100 for over a year now. On day one I uploaded the `2.18` version of this software. Has been working fine ever since.

Today I wanted to update it to the latest, but I can't as when I plug it in the PC (Windows 11 64bit), the drive created is only 8kB in size and (of course) like this the new firmware wouldn't fit on it. When I plug it in the USB (holding down the button) it says: `DFU:3.45`

This is how it looks in Windows:
<img width="271" alt="image" src="https://github.com/Ralim/IronOS/assets/12509879/67fbd342-5e0d-4b36-b348-96c84a306915">

I have looked through the discussions and bugs, but I only found similar issues, where the solution was to hold down the button when connecting the iron to Windows.

Any idea what's going on? How to get back the proper size?
Thank in advance!

**Comment by ia** [comment, marked as answer]
Hello. I don't have the solution unfortunately. Plus, there may be some important details which are missed / not mentioned. So I can provide only a few suggestions (i.e., that's what I would try - it's trivial & obvious but I would try it anyway):
- do not use USB hub while connecting to PC nor front USB panel (if it's "tower"-like PC);
- try different cables / ports on your PC while plugging-in;
- try to connect `TS100` in `dfu mode` to other PC(s) with other OS(es) if possible (at least just to check if the issue appears regardless OS / computer).
Maybe this help, maybe not.

**Reply by viktak** [reply]
Yeah, it worked on another PC...
I should have know....
Thanks!!!
